css/general.css

```diff
@@ -36,4 +36,20 @@ body::contextmenu {
 
 #options {
     margin: 10px 0 10px 0;
+}
+
```

Comment:
This looks more formal. Did this fix it on the i.o. on non-edge browsers?

Reply:
That part may have looked new because I accidentally re-formatted it. However, the only part I actually added was this:

`.crosshair-cursor-unclicked {
    cursor: url('../sprites/crosshair_unclicked.png') 16 16, crosshair;
}

.crosshair-cursor-clicked {
    cursor: url('../sprites/crosshair_clicked.png') 16 16, crosshair;
}

.pointer-cursor-unclicked {
    cursor: url('../sprites/pointer_unclicked.png') 10 4, pointer;
}

.pointer-cursor-clicked {
    cursor: url('../sprites/pointer_clicked.png') 10 4, pointer;
}`

Reply:
This is part of the alternate implementation of the crosshair/pointer.